frontend/packages/dev-console/integration-tests/support/pages/monitoring/monitoring-page.ts

```diff
@@ -0,0 +1,95 @@
+import { monitoringTabs } from '../../constants/monitoring';
+import { monitoringPO } from '../../pageObjects/monitoring-po';
+
+export const detailsPage = {
+  isTabSelected: (tabSelector: string) =>
+    cy
+      .get(tabSelector)
+      .parent('li')
+      .should('have.class', 'active'),
+  selectTab: (tabSelector: string) => cy.get(tabSelector).click(),
+};
+
+export const monitoringPage = {
+  dashboard: {
+    selectWorkload: (workloadName: string = 'All Workloads') => {
+      cy.get(monitoringPO.dashboardTab.workloadsFilter).click();
+      cy.byLegacyTestID('dropdown-text-filter').type(workloadName);
+      cy.get(`[id="${workloadName}-link"]`).click();
+    },
+    verifySection: () => cy.get(monitoringPO.dashboardTab.dashboard).should('be.visible'),
+    verifyCpuUsageGraph: () => {},
+    verifyMemoryUsageGraph: () => {},
+    verifyReceiveBandwidthGraph: () => {},
+    verifyTransmitBandwidthGraph: () => {},
+    verifyRateOfReceivedPacketsGraph: () => {},
+    verifyRateOfTransmittedPacketsGraph: () => {},
+    verifyRateOfReceivedPacketsDroppedGraph: () => {},
+    verifyRateOfTransmittedPacketsDroppedGraph: () => {},
+  },
+  metrics: {
+    enterQuery: (query: string) => {
+      cy.get(monitoringPO.metricsTab.queryExpression).type(query);
+    },
+    selectQuery: (queryValue: string) => {
+      cy.selectValueFromAutoCompleteDropDown(monitoringPO.metricsTab.selectQuery, queryValue);
+    },
+    verifyGraph: () => {
+      cy.get(monitoringPO.metricsTab.cpuGraph).should('be.visible');
+    },
+    verifyResetZoom: () => cy.byButtonText('Reset Zoom').should('be.visible'),
+    clickResetZoom: () => cy.byButtonText('Reset Zoom').click(),
+  },
+  alerts: {
+    search: (name: string) => {
+      cy.byLegacyTestID('item-filter').type(name);
+    },
+    filter: (
+      alertState: string[] = new Array('firing'),
+      severity: string[] = new Array('Critical'),
+    ) => {
+      cy.byLegacyTestID('filter-dropdown-toggle').click();
+      cy.get(`[data-test-row-filter="${alertState}"]`).click();
+      //  To Do
+      cy.byLegacyTestID(`[data-test-row-filter="${severity}"]`).click();
+    },
+    clickFilter: () => cy.byLegacyTestID('filter-dropdown-toggle').click(),
+  },
+  events: {
+    selectResources: (resourceName: string) => {
+      cy.selectValueFromAutoCompleteDropDown(monitoringPO.eventsTab.resources, resourceName);
+    },
+    selectType: () => {
+      cy.byLegacyTestID('dropdown-button').click();
+      cy.get('#all-link').click();
+      // To Do
```

Comment:
nit: we can remove it?

Reply:
The function is still not ready, but it's required so added it as as placeholder

Reply:
okay , will be good to add at least what's expected as part of TODO

Reply:
updated with the function itslef